src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -89,7 +89,7 @@ internal abstract class BrokerClient : IBrokerClient {
             await TaskUtilities.SwitchToBackgroundThread();
 
             try {
-                bool sessionExists = await IsSessionRunningAsync(name, cancellationToken);
+                bool sessionExists = false; //await IsSessionRunningAsync(name, cancellationToken);
```

Comment:
Yes, that is the intent (since we decided to not go with reconnection in RTM). Comment is just so that it's readily obvious how to revert this later, when we get to it.